feature-configs/demo/dpdk/is_ready.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+oc -n dpdk-testing get build -l app=s2i-dpdk
+
+oc -n dpdk-testing wait build --all --for condition=Complete --timeout 1s
```

Comment:
would it make sense to wait for the only dc we care of (`s2i-dpdk`)?

Reply:
The buildConfig doesn't have a condition field this is why I need to look onto the build and the build have a running number this is why I just look on all of them.